docs/security-sandboxing.md

```diff
@@ -1,3 +1,17 @@
 # Sandboxing
 
-... more coming soon
+One of WebAssembly (and Wasmtime's) main goals is to execute untrusted code in
+a safe manner inside of a sandbox. WebAssembly is inherently sandboxed by design
+(must import all functionality, etc), and Wasmtime plans to grow its own
+sandboxing mechanisms over time. This document is intended to cover the various
+sandboxing strategies that Wasmtime has as they are developed.
```

Comment:
> Wasmtime plans to grow its own sandboxing mechanisms over time

This sounds a little like we might be planning non-standard extensions to wasm. Could you expand on this perhaps by saying that different sandboxing implementation techniques come with different tradeoffs in terms of performance and feature limitations, and Wasmtime plans to offer users choices of which tradeoffs they want to make?